tmt/utils.py

```diff
@@ -3706,7 +3759,14 @@ def _normalize_string_list(
         if value is None:
             return []
 
-        return [value] if isinstance(value, str) else value
+        if isinstance(value, str):
+            return [value]
+
+        if isinstance(value, (list, tuple)):
+            return [item for item in value]
+
+        raise SpecificationError(
+            f"Field can be either string or list of strings, '{type(value)}' found.")
```

Comment:
Could we modify the normalization calls to accept the `keyname` as well? It would allow to mention the key in the erorr message. Currently we get:

```
Failed to load step data for PrepareInstallData: Field can be either string or list of strings, '<class 'bool'>' found.
The exception was caused by the previous exception:
Field can be either string or list of strings, '<class 'bool'>' found.
```

From this message user cannot see which field has incorrect value. I guess this would probably be material for a separate pull request.

Reply:
I believe we can, good point. I'd do that in a standalone PR.

Reply:
Ack.